Add close button to filter popover

ralphjsmit opened the pull request:
This PR adds a Close button to the filters popover, because some people do not know how to close the filter popover.

Example: 

<img width="581" alt="Schermafbeelding 2022-02-24 om 17 16 21" src="https://user-images.githubusercontent.com/59207045/155563601-8dfbf0e3-008c-455c-96eb-1775ba3acd69.png">

PS: I figured out that I could add the new translation key as well to the Dutch translation file. So I compared the Dutch file to the English one and updated all missing keys.

wychoong committed:
extract create modal actions

wychoong committed:
chore: styling

wychoong committed:
Update CanCreateRecords.php

wychoong committed:
Merge branch 'extract-create-modal-actions' of https://github.com/wychoong/filament into extract-create-modal-actions

wychoong committed:
Update CanCreateRecords.php

wychoong committed:
chore: styling

wychoong committed:
Merge branch 'extract-create-modal-actions' of https://github.com/wychoong/filament into extract-create-modal-actions

ryangjchandler committed:
feature: support custom global search providers

jacksleight committed:
feature: add prefix/postfix support to select fields

jacksleight committed:
Prevent multi-select word wrap

jacksleight committed:
Allow `HtmlString` objects

jacksleight committed:
Add url option

jacksleight committed:
trait CanOpenUrl trait

jacksleight committed:
Tweak compact file upload layout

jacksleight committed:
Add disable preview option

jacksleight committed:
Remove dark mode label background

jacksleight committed:
Reduce text size

jacksleight committed:
Add compact multiple item spacing

jacksleight committed:
Set min-height correctly in avatar layout

jacksleight committed:
Tweak

saade committed:
add filepond-plugin-media-preview

saade committed:
fix: adds package as dev dependency

jacksleight committed:
Merge branch 'laravel-filament:2.x' into file-upload-tweaks

jacksleight committed:
Merge branch 'feature/filepond-media-preview' into file-upload-tweaks

jacksleight committed:
Apply `disablePreview` to video and audio

jacksleight committed:
Fix color of drag and drob blob in dark mode

pxlrbt committed:
Allow closure customization for `columnSpan()`

martin-ro committed:
Update SpatieMediaLibraryFileUpload.php

Allow to define a conversion of image to be displayed

pxlrbt committed:
feat: Allow customisation for BelongsToSelect option labels

pxlrbt committed:
docs: Document `getOptionLabelFromRecordUsing()` function on BelongsToSelect

pxlrbt committed:
chore: styling

ap1969 committed:
Added alignment config to table row actions.

mohamedsabil83 committed:
[RTL] Fix padding of datetime picker

wychoong committed:
Merge branch '2.x' into extract-create-modal-actions

ManojKiranA committed:
Extract notification message

ManojKiranA committed:
remove undefined var

wychoong committed:
Merge branch '2.x' into extract-create-modal-actions

ManojKiranA committed:
pass argments to functions

wychoong committed:
extract attach records & rename methods

jacksleight committed:
Default to compact panel layout

wychoong committed:
global disable rel. manager create/attach another

wychoong committed:
global disable create resource create another

wychoong committed:
chore: styling

wychoong committed:
update doc & rename method

wychoong committed:
Merge branch 'extract-create-modal-actions' of https://github.com/wychoong/filament into extract-create-modal-actions

wychoong committed:
chore: styling

marcbelletre committed:
Fix select listboxes z-index

dmandrade committed:
Add filepond locale

mohamedsabil83 committed:
Apply phrase translations

ralphjsmit committed:
Add close button to filter popover

ralphjsmit committed:
Add close button to filter popover

ralphjsmit committed:
Update dutch translation file

ralphjsmit committed:
Update dutch translation file

danharrin committed:
wip

danharrin committed:
Merge branch 'pr/1653' into 2.x

danharrin committed:
wip

danharrin committed:
Merge branch 'pr/1702' into 2.x

danharrin committed:
wip

danharrin committed:
Merge branch 'pr/1672' into 2.x

danharrin committed:
Merge branch '2.x' into pr/1667

danharrin committed:
wip

danharrin committed:
Merge branch 'pr/1661' into 2.x

danharrin committed:
Update SpatieMediaLibraryFileUpload.php

danharrin committed:
Merge pull request #1654 from martin-ro/patch-7

Update SpatieMediaLibraryFileUpload.php

danharrin commented:
I think this should be an X button in the top right corner of the modal instead of a link button?

danharrin committed:
Update file-upload.css

danharrin committed:
Merge pull request #1626 from jacksleight/file-upload-tweaks

Tweak file upload layout, add filepond-plugin-media-preview, add disable preview option

danharrin committed:
wip

danharrin committed:
Merge branch 'pr/1611' into 2.x

danharrin committed:
Update select.blade.php

danharrin committed:
Merge branch '2.x' of https://github.com/laravel-filament/filament into 2.x

danharrin committed:
Update select.blade.php

danharrin committed:
Update file-upload.blade.php

danharrin committed:
Update file-upload.blade.php

danharrin committed:
Merge pull request #1689 from dmandrade/locale-filepond

Locale filepond

danharrin committed:
wip

ManojKiranA committed:
Show notification if only available

ManojKiranA committed:
format if condition

ManojKiranA committed:
add space after if

ManojKiranA committed:
conditional notification on all pages

pathros committed:
Added new missing Spanish translations.

Fixed minor details in Spanish translations.

danharrin committed:
Merge pull request #1717 from pathros/add-missing-spanish-translations

Added new missing Spanish translations.

danharrin committed:
chore: styling

panakour committed:
add missing Exception class

danharrin committed:
wip

danharrin committed:
Merge pull request #1718 from panakour/fix/add-missing-exception-class

Add missing Exception class

ryangjchandler committed:
docs: return navigation builder

ryangjchandler committed:
chore: styling

ryangjchandler committed:
tests: GlobalSearch

ryangjchandler committed:
Merge branch 'feature/global-search-provider' of github.com:laravel-filament/filament into feature/global-search-provider

ryangjchandler committed:
chore: styling

danharrin committed:
Merge pull request #1721 from laravel-filament/docs/navigation-api

docs: return navigation builder

danharrin committed:
chore: build assets

ryangjchandler committed:
refactor: introduce search results builder and DTO

ryangjchandler committed:
chore: styling

jeffgreco13 committed:
feat: Add custom tags to filament's header

Adds a method registerMeta for adding custom <meta>, <link>, or any other tag to the filament <head>

Commit:
Merge branch 'laravel-filament:2.x' into 2.x

ryangjchandler committed:
refactor: eliminate old array format for results

ryangjchandler committed:
chore: styling

jeffgreco13 committed:
Revert "feat: Add custom tags to filament's header"

This reverts commit eb6d9b1a9422db2dcb1c43a00bd1c7d4986a6d63.

ryangjchandler committed:
chore: touch ups

ryangjchandler committed:
chore: remove admin config changes

ryangjchandler committed:
fix: simple resources showing delete action for all records

ryangjchandler committed:
fix: hide edit action for simple resources if cannot edit

ryangjchandler committed:
fix: hide create action if cannot create records

ryangjchandler committed:
fix: remove record arg from canCreate check

jeffgreco13 committed:
Merge branch '2.x' of https://github.com/jeffgreco13/filament into 2.x

jeffgreco13 committed:
fix: docs styling

jeffgreco13 committed:
fix: indent

danharrin committed:
feature: Table record actions

danharrin committed:
chore: styling

danharrin committed:
Merge pull request #1735 from laravel-filament/feature/table-record-actions

feature: Table record actions

danharrin committed:
chore: build assets

danharrin committed:
wip

danharrin committed:
Update CanEditRecords.php

danharrin committed:
Merge branch 'fix/simple-resource-delete-checks' into 2.x

danharrin committed:
chore: build assets

danharrin committed:
Update tables.php

danharrin committed:
Update 05-actions.md

danharrin committed:
Update 05-actions.md

danharrin committed:
Update actions-cell.blade.php

danharrin committed:
Merge branch 'pr/1659' into 2.x

danharrin committed:
wip

danharrin committed:
Merge pull request #1605 from laravel-filament/feature/global-search-provider

feature: support custom global search providers

danharrin committed:
chore: build assets

danharrin committed:
wip

danharrin committed:
Update CreateRecord.php

danharrin committed:
Merge branch 'pr/1711' into 2.x

danharrin committed:
chore: styling

drhoussem commented:
@danharrin I personally think it's better to put it next to reset filter from a UX perspective

a21ns1g4ts committed:
Add pt_BR translations - Settings plugin

danharrin committed:
Merge pull request #1738 from a21ns1g4ts/add-pt_BR-settings-plugin

Add pt_BR translations - Settings plugin

jeffgreco13 committed:
update: requested changes

mallardduck committed:
Fix bug

danharrin committed:
Merge pull request #1741 from mallardduck/patch-1

Fix bug #1740

FurkanGM committed:
fix: responsive compatibility problem on table filters

FurkanGM committed:
table filter now always positing on the right

FurkanGM committed:
revert table filter popover classes

RVxLab committed:
Add Dutch translations for settings and translatable plugins

mallardduck committed:
make sure phpstan is in dev deps

mallardduck committed:
Adjust types for PHPstan

mallardduck committed:
Add union return to better typehint

mallardduck committed:
Ensure correct query type is passed to orderBy

mallardduck committed:
derp...larastan is already here

mallardduck committed:
Correct bootstrap for Orchestra 6.*

mallardduck committed:
reset lock file back to before this PR

wychoong committed:
add middlewares to page and resource

danharrin committed:
Merge pull request #1744 from RVxLab/2.x

Add Dutch translations for settings and translatable plugins

danharrin committed:
Merge pull request #1750 from wychoong/add-middleware-to-page-&-resource-routes

add middlewares to page and resource

danharrin committed:
Merge branch 'pr/1743' into 2.x

danharrin committed:
wip

danharrin committed:
chore: styling

danharrin committed:
Update app.php

danharrin committed:
wip

danharrin committed:
wip

danharrin committed:
Merge pull request #1749 from mallardduck/enable-PHPstan

Refactors to enable PHPStan

danharrin committed:
chore: styling

danharrin committed:
Update phpstan.yml

danharrin committed:
Update 07-appearance.md

danharrin committed:
Merge pull request #1722 from jeffgreco13/2.x

feat: Add ability to add custom tags to the head

pixelpeter committed:
Update section background-color to white

danharrin committed:
Merge branch '2.x' into pr/1516

danharrin committed:
wip

danharrin committed:
chore: styling

danharrin committed:
Update CanAttachRecords.php

danharrin committed:
Merge branch 'extract-create-modal-actions' of https://github.com/wychoong/filament into pr/1516

danharrin committed:
Update RelationManager.php

danharrin committed:
Update 02-resources.md

danharrin committed:
Merge pull request #1516 from wychoong/extract-create-modal-actions

extract create modal actions

danharrin committed:
Merge branch '2.x' into pr/1658

danharrin committed:
Update BelongsToSelect.php

danharrin committed:
Update BelongsToSelect.php

danharrin committed:
Update 03-fields.md

danharrin committed:
Add to other fields

danharrin committed:
extra select input attributes

danharrin committed:
Update composer.lock

pixelpeter committed:
ADD: use same class as card component for dark mode

danharrin committed:
Update select.blade.php

FurkanGM committed:
fix: wrong slug generation on pages

danharrin committed:
wip

danharrin committed:
Merge branch '2.x' of https://github.com/FurkanGM/filament into pr/1743

danharrin committed:
wip

danharrin committed:
Update Page.php

danharrin committed:
Merge pull request #1759 from FurkanGM/2.x

fix: wrong slug generation on pages

danharrin committed:
wip

danharrin committed:
Merge pull request #1751 from pixelpeter/2.x

Update section background-color to white

danharrin committed:
Update index.blade.php

danharrin committed:
chore: Switch 404 misconfigured admin panel to 403

danharrin committed:
Merge pull request #1763 from laravel-filament/chore/403-admin-panel

chore: Switch 404 misconfigured admin panel to 403

danharrin committed:
chore: build assets

danharrin committed:
fix: Bulk actions firing twice

danharrin committed:
Merge pull request #1764 from laravel-filament/fix/bulk-actions-firing-twice

fix: Bulk actions firing twice

danharrin committed:
chore: build assets

danharrin committed:
wip

danharrin committed:
Merge pull request #1765 from laravel-filament/fix/select-value-type

fix: Select value type

danharrin committed:
chore: build assets

jacksleight committed:
Fix searchable select width

danharrin committed:
Merge pull request #1769 from jacksleight/fix-select

Fix searchable select width

jacksleight committed:
Allow HtmlString objects in stat card label and description values

danharrin committed:
feature: Stats overview polling

danharrin committed:
Merge pull request #1770 from laravel-filament/feature/stats-overview-polling

feature: Stats overview polling

danharrin committed:
chore: build assets

jacksleight committed:
Allow entire stats card to be linkable

jacksleight committed:
Tweak echos

danharrin committed:
Merge pull request #1771 from jacksleight/link-cards

Allow stat cards to be linked, allow `HtmlString` objects in stat card labels and descriptions

danharrin committed:
chore: styling

danharrin committed:
feature: Indicate selected table rows

danharrin committed:
Merge pull request #1772 from laravel-filament/feature/table-row-selection-indication

feature: Indicate selected table rows

danharrin committed:
wip

danharrin committed:
Update card.blade.php

danharrin committed:
feature: Register forms from traits

danharrin committed:
refactor

danharrin committed:
Merge pull request #1773 from laravel-filament/feature/register-forms-from-traits

feature: Register forms from traits

danharrin committed:
chore: build assets

ralphjsmit committed:
Merge branch 'add-popover-close-button' of https://github.com/ralphjsmit/filament into add-popover-close-button

ralphjsmit committed:
Convert to close button

ralphjsmit committed:
Convert to close button

ralphjsmit committed:
Remove 'Close' link

ralphjsmit committed:
Merge branch '2.x' into add-popover-close-button

ralphjsmit committed:
Remove spacing
